core/src/main/scala/geotrellis/store/GeoTrellisRasterSourceLegacy.scala

```diff
@@ -0,0 +1,159 @@
+/*
+ * Copyright 2019 Azavea
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package geotrellis.store
+
+import geotrellis.proj4._
+import geotrellis.raster.io.geotiff.{Auto, AutoHigherResolution, Base, OverviewStrategy}
+import geotrellis.raster.resample.{NearestNeighbor, ResampleMethod}
+import geotrellis.raster._
+import geotrellis.layer._
+import geotrellis.vector._
+
+/**
+  * TODO: remove after upgrading up to GeoTrellis 3.3.0
```

Comment:
Do we have an issue in this project to upgrade to GT 3+? If not, lets create one. And once we have an issue, can you note in it that we should be removing these legacy classes?

Reply:
@CloudNiner this project is already updated up to the latest GT; this one porbably to upgrade exactly up to the next GT release https://github.com/locationtech/geotrellis/pull/3180

Reply:
https://github.com/geotrellis/geotrellis-server/issues/197